docs/INSTALLATION.md

```diff
@@ -25,161 +46,144 @@ De opbouw van dit certificaat is als volgt:
 
 Het private certificate dient los als .key bestand opgeslagen te worden.
 
+### Tools
+
+Installeer de volgende tools:
 
-#### Tools
 - [kubectl](https://kubernetes.io/docs/tasks/tools/)
 - [Helm](https://helm.sh/docs/intro/install/)
 - [Haven](https://haven.commonground.nl/techniek/compliancy-checker)
 - [Powershell](https://learn.microsoft.com/en-us/powershell/)
 
-#### Haven
-Er dient een haven-compliant cluster ingericht te worden.
-Hiervoor zijn referentie-implementaties te vinden op de [haven](https://haven.commonground.nl/techniek/aan-de-slag) site.
+### Haven
+
+Zorg voor een haven-compliant cluster. Referentie-implementaties zijn te vinden op de [Haven](https://haven.commonground.nl/techniek/aan-de-slag) site.
 
-#### Azure
-Bij het volgen van de [Azure](https://haven.commonground.nl/techniek/aan-de-slag/azure) referentie dient met de volgende extra zaken rekening gehouden te worden.
+### Azure
 
-- minimaal 3 nodes
-- high availability aan zetten
-- local logins met RBAC gebruiken, GEEN Azure Active Directory!
-- Kubernetes version 1.25.4 gebruiken
+Bij het volgen van de [Azure](https://haven.commonground.nl/techniek/aan-de-slag/azure) referentie:
 
-Per servertype is er een quota aan het maximale aantal cores dat gebruikt mag worden in de regio.
-Als dit overschreden wordt, dan kan dit in het quota scherm aangepast worden:
-[https://portal.azure.com/#view/Microsoft_Azure_Capacity/QuotaMenuBlade/~/myQuotas](https://portal.azure.com/#view/Microsoft_Azure_Capacity/QuotaMenuBlade/~/myQuotas).
+- Minimaal 3 nodes
+- High availability aanzetten
+- Local logins met RBAC gebruiken, GEEN Azure Active Directory!
+
+Meer informatie over quota aanpassingen: [Azure Quota](https://portal.azure.com/#view/Microsoft_Azure_Capacity/QuotaMenuBlade/~/myQuotas).
+

```

Comment:
Waarom is het nodig om hier naar informatie over quota aanpassingen te verwijzen?